data/json/recipes/other/tool.json

```diff
@@ -134,9 +134,8 @@
     "type": "recipe",
     "category": "CC_OTHER",
     "subcategory": "CSC_OTHER_TOOLS",
-    "skill_used": "fabrication",
-    "difficulty": 4,
-    "skills_required": [ "fabrication", 4 ],
+    "skill_used": "electronics",
+    "difficulty": 3,
```

Comment:
So let's compare: 

- coffeemaker: needs recipe books, needs only tool with `screw` quality, needs 10 charges of toolset or soldering iron, needs much more components, including `amplifier` and `element`, but no SEM.

- food processor: no need for recipe books, needs tools with `screw` and `hammer` qualities, needs 20 charges of toolset or soldering iron, needs much less components, but including SEM.